recipes/libhdbpp/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "libhdbpp" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/tango-controls/hdbpp/libhdbpp/-/archive/v{{ version }}/libhdbpp-v{{ version }}.tar.gz
+  sha256: 53e1647d0106787adaed5d9e82c3eb53c5d5b1a4892387b1e52a183fadde4b93
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  # Library doesn't link against cpptango (only headers used)
+  ignore_run_exports:
+    - cpptango
+
```

```suggestion
  run_exports:
    - {{ pin_subpackage('libhdbpp') }}

```
People that build against this library may not need cpptango, but they definitely need this library? I assume this project uses SEMVER?

---

Good point. Thanks! Added.